examples/motoko_examples/persistent-storage/canisters/azle/index.ts

```diff
@@ -0,0 +1,57 @@
+import { ic, Init, nat, nat64, Opt, Query, Stable, Update } from 'azle';
+
+type PerfResult = {
+    wasm_body_only: nat64;
+    wasm_including_prelude: nat64;
+};
+
+let perf_result: Opt<PerfResult> = null;
+
+export function get_perf_result(): Query<Opt<PerfResult>> {
+    return perf_result;
+}
+
+function record_performance(start: nat64, end: nat64): void {
+    perf_result = {
+        wasm_body_only: end - start,
+        wasm_including_prelude: ic.performance_counter(0)
+    };
+}
+
+type StableStorage = Stable<{
+    counter: nat;
+}>;
+
+export function init(): Init {
```

Comment:
As a best practice let's pull the `ic.stable_storage` up into a variable called `stable_storage` and use that to access the stable storage, like `stable_storage.counter = 0n`